package.json

```diff
@@ -2,8 +2,8 @@
     "name": "vscode-clangd-multi-project",
```

Comment:
Ad the name and display name is something we likely don't want to ever change, maybe we should be more generic here from the start? E.g. `vscode-clangd-cdtcloud`?